proarc-webapp/src/main/java/cz/cas/lib/proarc/webapp/server/rest/DigitalObjectResource.java

```diff
@@ -1178,6 +1143,31 @@ public Response getDissemination(
         return new SmartGwtResponse<Map<String,Object>>(Collections.singletonMap("processId", (Object) 0L));
     }
 
+    @DELETE
+    @Path(DigitalObjectResourceApi.DISSEMINATION_PATH)
+    @Produces({MediaType.APPLICATION_JSON})
+    public SmartGwtResponse deleteDissemination(
+            @QueryParam(DigitalObjectResourceApi.DIGITALOBJECT_PID) String pid,
+            @QueryParam(DigitalObjectResourceApi.BATCHID_PARAM) String batchId,
```

Comment:
Proč není batchId Integer jako v ostatních případech?

Reply:
Jde o stav, kdy je `batchId` v GWT `null`, v rámci požadavku na backend framework nezašle `null`, ale `"null"` (instanceof String). Tedy volání endpointu selhalo kvůli nekompatibilnímu typu String -> Integer.

Reply:
V tomto případě je potřeba odstranit/nenastavovat patřičný null attribute ze SmartGWT DELETE requestu. Viz. MediaEditor.removeDS(). Kvůli této SmartGWT podivnosti bych neohýbal serverovou část.

Reply:
MediaEditor upraven aby nenastavoval batchId do query v případě že je hodnota null.